nf_core/lint/nextflow_config.py

```diff
@@ -262,6 +268,44 @@ def nextflow_config(self):
                 )
             )
 
+    # Check if custom profile params are set correctly
+    if self.nf_config.get("params.custom_config_version", "").strip("'") == "master":
+        passed.append("Config ``params.custom_config_version`` is set to ``master``")
```

Comment:
These are used for log output with the `rich` library, not .rst format like the doc string / readtbedocs. These look best with just single quotes.